dnf5/commands/repoquery/repoquery.cpp

```diff
@@ -79,6 +82,13 @@ void RepoqueryCommand::set_argument_parser() {
     installed->set_const_value("true");
     installed->link_value(installed_option);
 
+    auto userinstalled = parser.add_new_named_arg("userinstalled");
+    userinstalled->set_long_name("userinstalled");
+    userinstalled->set_description("Limit to packages that were installed by user.");
```

Comment:
I am sorry, but I would woild like to modify the description. I checked the implementation and it correctly returns all packages that are not installed as dependency or weak dependency. It means everything what was installed by user request, or indirectly as a part of module profile or comps group. Additionally it returns as user installed packages packages with unknown reason.

I think that it is correct, then the description and may be even name of the option does not fit.

Reply:
@j-mracek I changed the description and the documentation.